styles/style.css

```
@@ -0,0 +1,162 @@
+body {
+  margin: 0px;
+  font-family: 'Merriweather', serif;
+  width: 100%
+}
+
+/* nav bar has three columns */
+nav ul {
```

Nice use of css selectors!

`this_is_a_variable`
```suggestion
nav ul li {
```